app/src/pages/ExploreDataPage.tsx

```diff
@@ -88,8 +87,31 @@ function ExploreDataPage() {
     activeSelections: defaultValuesWithOverrides,
   });
 
+  const [sticking, setSticking] = useState<boolean>(false);
+
+  useEffect(() => {
+    const header = document.getElementById("ExploreData");
+    const stickyBarOffsetFromTop: number = header ? header.offsetTop : 1;
+    const scrollCallBack: any = window.addEventListener("scroll", () => {
+      if (window.pageYOffset > stickyBarOffsetFromTop) {
+        if (header) {
```

Comment:
What's the situation where this isn't present?

Reply:
If someone deleted the div that contains the ID... it was mostly just to address a type error